platform-sdk/docs/core/wiringDiagramLink.md

```diff
@@ -0,0 +1,6 @@
+
+When making any change that modifies the wiring diagram,
+please regenerate the diagram and update this page with the new diagram.
```

Comment:
Minor: this may be a tough ask that's easy to miss/forget about, especially as time progresses. Also, this isn't much of a creative work that strictly requires a human to do this manually. In an ideal world, the diagram would be auto-generated from the wiring code.

Reply:
Sounds good. Perhaps this note in `wiringDiagramLink.md` could provide some instructions on how to re-generate the diagram by that program then.